### PR TITLE
Update "st2 execution tail" CLI command to just print raw action output by default

### PR DESCRIPTION
**Pull request by Kami**
This pull request updates `st2 execution tail` command to just print raw output by default. This way it's consistent with the `/output` API endpoint.

User can also request metadata (timestamp, output type) to be printed by using `--include-metadata` flag.

Without the flag (default behavior):

```bash
st2 execution tail last
stderr -> Line: 1
stdout -> Line: 2
stderr -> Line: 3
stdout -> Line: 4
stderr -> Line: 5
stdout -> Line: 6
stderr -> Line: 7
stdout -> Line: 8
stderr -> Line: 9
stdout -> Line: 10

Execution 59ba7e5d0640fd4f73312076 has completed.
```

With the flag (previous default behavior):

```bash
st2 execution tail last --include-metadata
[2017-09-14T13:07:07.229150Z][stderr] stderr -> Line: 1
[2017-09-14T13:07:08.234801Z][stdout] stdout -> Line: 2
[2017-09-14T13:07:09.235263Z][stderr] stderr -> Line: 3
[2017-09-14T13:07:10.237950Z][stdout] stdout -> Line: 4
[2017-09-14T13:07:11.238072Z][stderr] stderr -> Line: 5
[2017-09-14T13:07:12.239523Z][stdout] stdout -> Line: 6
[2017-09-14T13:07:13.240308Z][stderr] stderr -> Line: 7
[2017-09-14T13:07:14.241287Z][stdout] stdout -> Line: 8
[2017-09-14T13:07:15.240921Z][stderr] stderr -> Line: 9
[2017-09-14T13:07:16.242700Z][stdout] stdout -> Line: 10

Execution 59ba7ef80640fd4f73312079 has completed.
```

### CODE REVIEW

**Comment by Kami** [comment]
I will also make the corresponding change on the API endpoint side - add `?include_metadata=True` query parameter.

**Comment by lakshmi-kannan** [comment]
LGTM.

File: st2client/st2client/commands/action.py [hunk 1]
```diff
@@ -1379,6 +1384,7 @@ def run(self, args, **kwargs):
     def run_and_print(self, args, **kwargs):
         execution_id = args.id
         output_type = getattr(args, 'output_type', None)
+        include_metadata = args.include_metadata
```
Comment (lakshmi-kannan) [inline st2client/st2client/commands/action.py]: The metadata is simply timestamp, right? So should we just say --with-timestamp? Just a thought.

Comment (Kami) [reply st2client/st2client/commands/action.py]: It's timestamp + stream / output type.

And we also store some other metadata on the model which we might want to include in the future so perhaps better to keep it as it is.

**Pull request merged**